pkg/networkmanager/interfaces.js

```diff
@@ -2597,6 +2621,12 @@ PageNetworkInterface.prototype = {
                 }
             }
 
+            function render_zone_row() {
```

Comment:
[I am attaching a global comment here in order to turn it into a review comment]

We need to also do something sensible when FirewallD is not available, such as with Debian, Ubuntu, and the Atomics.

I guess it will be best to hide the "Zone" setting completely in that case.